server/controllers/courses/index.js

```diff
@@ -0,0 +1,28 @@
+const router = require('express').Router();
+
+const { courses } = require('./courses');
+const { course } = require('./course');
+const { coursesLevel } = require('./level');
+const { studentCourses } = require('./sudent');
+const { applyCourse } = require('./postApplyCourse');
+const { addCourse } = require('./addCourse');
+const { deleteCourse } = require('./deleteCourse');
+
+router
+  .route('/')
+  .get(courses)
+  .post(addCourse);
+
+router
+  .route('/:id')
+  .get(course)
+  .put(applyCourse)
+  .delete(deleteCourse);
+
+router
+  .use('/level', coursesLevel);
+
+router
+  .use('/student', studentCourses);
```

Comment:
@AbdallahAmmar96 This isn't a big deal to worry about right now, more of a suggestion for the future: I think the RESTful convention for urls to express relationships between items is to put the "owned" item after the "owner", in the url path, so for example to get all of a student's courses, REST url conventions would say that the url is like this:
`GET /students/:studentId/courses` - the courses come after the student in the url, because you want the courses for that student.
.. then that would fit nicely with other urls for that student:
`GET /students/:studentId` - info for that student
`GET /students/:studentId/attendance` - attendance records for that student
and so on

Reply:
I will consider it